.githooks/pre-commit

```diff
@@ -119,6 +119,6 @@ if command_exists "colcon"; then
     fi
 fi
 
-cd -
+cd /
```

Comment:
This should still be `cd -` (return to previous directory, i.e., out of the `ros` directory) and it should be within the `if command_exists "colcon"` block

Reply:
When I tried cd - I was getting some weird error and online it said to just replace it with cd /. I'll look more into it next meeting

Reply:
`cd /` should put you into your computer's root, not the repo root, which isn't what we want (unless there is a difference between MacOS and Linux, but I doubt it).

If the error was `-bash: cd: OLDPWD not set`, that's because you hadn't `cd`'d anywhere before using `cd -` so there was nowhere to go back to. This will not occur if `cd -` is inside the colcon block because that is guaranteed to `cd` into the `ros` directory

Reply:
yep that was the error. I'll fix this soon.